src/message.rs

```diff
@@ -23,6 +23,18 @@ impl Message {
     pub fn insert_data(&mut self, field: &str, value: impl Into<PayloadValue>) {
         self.payload.insert(String::from(field), value.into());
     }
+
+    pub fn get_payload(&self) -> &HashMap<String, PayloadValue> {
```

Comment:
Getters should not have `get_` in the name. See https://doc.rust-lang.org/1.0.0/style/style/naming/README.html

Reply:
I was thinking the same, but didn't know if it would work but really cool that there's actually official guidelines for this!